tests/integration_tests/test_grpc_only.py

```diff
@@ -45,9 +47,6 @@ def grpc_eth_call(port: int, args: dict, chain_id=None, proposer_address=None):
     ).json()
 
 
-@pytest.mark.skip(
```

Comment:
@facs95 do you mind if I keep this, since I fail to reproduce when long run this test after seeing https://github.com/evmos/ethermint/pull/1510#issuecomment-1330629297, will be happy to help if there's bug hidden

Reply:
oh yes sorry - This was already a fix by @ramacarlucho so we can delete the skip. It should fail randomly anymore 🙏